hostprocess/calico/node/node-service.ps1

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 # Modified from https://github.com/projectcalico/node
-ipmo .\libs\calico\calico.psm1
-ipmo .\libs\hns\hns.psm1
+ipmo .\libs\calico\calico.psm1 -Force
+ipmo .\libs\hns\hns.psm1 -Force -DisableNameChecking
```

Comment:
Those are from the projectcalico/calico repository [node-service.ps1](https://github.com/projectcalico/calico/blob/master/node/windows-packaging/CalicoWindows/node/node-service.ps1#L18), perhaps those are not needed. Shall I remove them?

Reply:
thanks, this is probably because those scripts run in the context of the host where it might have already been imported. Since we are running inside the container I don't they are required but likely won't hurt anything.